source/extensions/access_loggers/common/grpc_access_logger.h

```diff
@@ -75,10 +75,10 @@ template <typename GrpcAccessLogger, typename ConfigProto> class GrpcAccessLogge
 
 template <typename LogRequest, typename LogResponse> class GrpcAccessLogClient {
 public:
-  GrpcAccessLogClient(Grpc::RawAsyncClientPtr&& client,
+  GrpcAccessLogClient(Grpc::RawAsyncClientSharedPtr client,
```

Comment:
pass shared_ptr as const ref. That saves an extra inc/dec of the ref-count in the shared_ptr, here and elsewhere in the PR.

Reply:
Thanks.